tests/test_time_array.py

```diff
@@ -75,7 +75,7 @@ class TestCallableTimeArray:
     @pytest.fixture(autouse=True)
     def _setup(self):
         f = lambda t: t * jnp.array([1, 2])
-        self.x = CallableTimeArray(f, ())
+        self.x = timecallable(f)
```

Comment:
Can you also replace calls to `ConstantTimeArray` with the `constant()` constructer?

Reply:
No, because the shape of the constant array is not legal (size `(n,)` instead of `(n,n)`), so it breaks tests.

Reply:
Ha ok. Maybe we should make these time arrays more general some day, but 👌 for now.